mypy/main.py

```diff
@@ -734,6 +734,13 @@ def add_invertible_flag(
         help="Disable strict Optional checks (inverse: --strict-optional)",
     )
 
+    add_invertible_flag(
```

Comment:
We don't need to show this in the mypy help output, since this is an internal option. You can use `help=argparse.SUPPRESS` to hide this.